content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,180 @@ aliases:
 ---
 
 Use the `influxd upgrade` command to upgrade InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
+The `upgrade` command exists provide an in-place upgrade path from InfluxDB 1.8.x to InfluxDB 2.0.x.
+
+<!--
+The upgrade process copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
 [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 to 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket).
+-->
+Specifically, the upgrade process does the following:
 
-{{% warn %}}
-Back up all data before upgrading with `influx upgrade`.
-{{% /warn %}}
+1. Reads existing the configuration file and generates a corresponding InfluxDB 2.0 configuration file with corresponding settings (where appropriate).
+   This file is placed `~/.influxdbv2/config.toml`.
+2. Upgrades the paths for metadata and the storage engine from current locations to `~/.influxdbv2/meta` and `~/.influxdbv2/engine` respectively (unless otherwise specified).
+3. Copies existing data and WAL files into [buckets](/influxdb/v2.0/reference/glossary/#bucket) in the new 2.0 storage engine.
+4. Creates required database and retention policy [(DBRP)](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) mappings required to access existing data via InfluxQL.
+5. Reads existing metadata and migrates non-admin users, passwords, and permissions into a 1.x authorization compatible store within `~/influxdbv2/influxdb.bolt`.
+
+After running `influxd upgrade`, when first starting InfluxDB 2.0 the database generates time series index (TSI) files.
+(This may take some time depending upon the volume of data present.)
+
+## Before you begin: important considerations
+
+Before upgrading to InfluxDB 2.0, please consider the following guidelines;
+some or all might apply to your specific installation and use case.
+The sections below contain our recommendations for addressing possible gaps in the upgrade process.
+You may find while reading that you'd like to hold off on upgrading for now.
+
+### Continuous queries
+
+if you are currently using continuous queries, we recommend extracting them from your InfluxDB 1.x instance with `show continuous queries` prior to upgrade.
+Run the following:
+
+{{< keep-url >}}
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
```

Comment:
Either put CQs in parentheses after continuous queries above or spell it out here.